.github/workflows/node.js.yml

```diff
@@ -29,6 +29,9 @@ jobs:
     - run: yarn install
     - run: yarn run build
     - run: yarn workspace @godwoken-web3/api-server run test tests/db/helpers.test.ts
+      env:
```

Comment:
please remove this line.

Reply:
I can't. Tests will fail.

```
➜  godwoken-web3 git:(feat/env-db-constants) ✗ yarn workspace @godwoken-web3/api-server run test tests/db/helpers.test.ts
yarn workspace v1.22.17
yarn run v1.22.17
$ ava tests/db/helpers.test.ts

  Uncaught exception in tests/db/helpers.test.ts

  Error: no env DATABASE_URL provided

  › getRequired (src/base/env-config.ts:35:11)
  › Object.<anonymous> (src/base/env-config.ts:7:16)
  › Module.m._compile (/Users/kuzi/projects/godwoken-web3/node_modules/ts-node/src/index.ts:1056:23)
  › require.extensions.<computed> (/Users/kuzi/projects/godwoken-web3/node_modules/ts-node/src/index.ts:1059:12)

  ✖ tests/db/helpers.test.ts exited with a non-zero exit code: 1
  ─

  1 uncaught exception
error Command failed with exit code 1.
```

I would have to change the way environment config works to add lazy loading (right now getRequired fails as soon as you import env config to other file and there's no DATABASE_URL or GODWOKEN_JSON_RPC env vars) to fix this, but it seems out of scope of this pull request.

Reply:
I see